ci-operator/step-registry/ipi/install/openstack/ipi-install-openstack-commands.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
Remove this file and use ipi-install-command.sh instead (just set the correct cluster_type)